discovery/src/main/java/org/wildfly/extension/discovery/AggregateDiscoveryProviderRegistrar.java

```diff
@@ -1,51 +0,0 @@
-/*
- * Copyright The WildFly Authors
- * SPDX-License-Identifier: Apache-2.0
- */
-package org.wildfly.extension.discovery;
-
-import java.util.Collection;
-import java.util.List;
-import java.util.function.Function;
-
-import org.jboss.as.controller.AttributeDefinition;
-import org.jboss.as.controller.OperationContext;
-import org.jboss.as.controller.OperationFailedException;
-import org.jboss.as.controller.PathElement;
-import org.jboss.dmr.ModelNode;
-import org.wildfly.discovery.impl.AggregateDiscoveryProvider;
-import org.wildfly.discovery.spi.DiscoveryProvider;
-import org.wildfly.subsystem.resource.ResourceDescriptor;
-import org.wildfly.subsystem.resource.capability.CapabilityReference;
-import org.wildfly.subsystem.resource.capability.CapabilityReferenceListAttributeDefinition;
-import org.wildfly.subsystem.service.ResourceServiceInstaller;
-import org.wildfly.subsystem.service.ServiceDependency;
-import org.wildfly.subsystem.service.capability.CapabilityServiceInstaller;
-
-/**
- * Registers the aggregate discovery provider resource definition.
- * @author Paul Ferraro
- */
-public class AggregateDiscoveryProviderRegistrar extends DiscoveryProviderRegistrar {
```

Comment:
Should not this be first deprecated and then removed?

Reply:
This is not a public module.